example/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 0.13.15
+sbt.version=1.2.0
```

Comment:
We should update to sbt version 1.2.1, which fixes a [forward binary compatibility breakage](https://developer.lightbend.com/blog/2018-08-07-sbt-1-2-1/) introduced in 1.2.0.

Reply:
The first part seems more important for plugins than for end users like us, but we were affected by the second point, regarding unknown configurations. Updated to SBT 1.2.1 in d53bc33.